appdata.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/packaging/org.freeorion.FreeOrion.metainfo.xml b/packaging/org.freeorion.FreeOrion.metainfo.xml
```

Comment:
Has it been upstreamed?

Reply:
https://github.com/freeorion/freeorion/pull/3146 & https://github.com/freeorion/freeorion/pull/3145